test/unit/org/apache/cassandra/service/ActiveRepairServiceTest.java

```diff
@@ -420,7 +420,8 @@ public void testRejectWhenPoolFullStrategy() throws InterruptedException
 
             // Submission is unblocked
             Thread.sleep(250);
-            validationExecutor.submit(() -> {});
+            validationExecutor.submit(() -> {
```

Comment:
It is ok to have lambdas on the same line via the style guide ([link](https://cassandra.apache.org/_/development/code_style.html#formatting)):

> { and } are placed on a new line except when empty or opening a multi-line lambda expression. Braces may be elided to a depth of one if the condition or loop guards a single expression.

> Lambda expressions accepting a single parameter should elide the braces that encapsulate the parameter. E.g. x → doSomething() and (x, y) → doSomething()

but change is fine too.